### PR TITLE
Feature/score

### PR DESCRIPTION
**Pull request by vitor-paholsky**

### CODE REVIEW

File: server/database/repository/score_repository.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	Save(score ScoreEntity) (string, error)
+}
+
+type ScoreEntity struct {
+	Score    float32    `json:"score"`
+	Id       string     `json:"id"`
+	RiderId  string     `json:"riderId"`
+	CreatedAt time.Time  `json:"createdAt"`
+	UpdatedAt *time.Time `json:"updatedAt,omitempty"`
+}
```
Comment (victor-schumacher) [inline server/database/repository/score_repository.go:21]: can you format this?
Id should be first?

File: server/database/repository/score_repository.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+type ScoreRepo struct {
+	database database.DbConnection
+}
+
+func NewScoreRepository(d database.DbConnection) ScoreRepo {
+	return ScoreRepo{database: d}
+}
+
+func (s ScoreRepo) Save(score ScoreEntity) (string, error) {
+
```
Comment (victor-schumacher) [inline server/database/repository/score_repository.go:32]: Can you rancar this space?

File: server/http/score/score.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+type Manager struct {
+	scoreRepository repository.Score
+}
+
+func NewHandler(score repository.Score) Manager {
+	return Manager{scoreRepository: score}
+}
+
+func (m Manager) Handle() http.HandlerFunc {
+
```
Comment (victor-schumacher) [inline server/http/score/score.go:23]: Can rancar this space?

**Comment by kaduartur** [comment]
LGTM!

File: server/http/score/score.go [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	if err != nil {
+		w.Write([]byte("Errow while processing data"))
+		return
+	}
+
+	scoreId, err := m.createScore(score)
+	if err != nil {
+		w.WriteHeader(http.StatusUnprocessableEntity)
+		w.Write([]byte("Error while processing data score"))
+
```
Comment (victor-schumacher) [inline server/http/score/score.go:49]: space

File: server/http/score/score.go [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+		return
+	}
+
+	w.Write([]byte(scoreId))
+}
+
+func (m Manager) createScore(s server.Score) (string, error) {
+
```
Comment (victor-schumacher) [inline server/http/score/score.go:57]: space

File: server/http/score/score.go [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+	scoreEntity := repository.ScoreEntity{
+		Score:     s.Score,
+		Id:        s.Id,
```
Comment (victor-schumacher) [inline server/http/score/score.go:60]: Why you need id to create?
Maybe you can remove this

**Pull request merged**